guidelines.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Package Guidelines
+description: Guidelines for Pangeo packages to follow
+---
+
+Our vision for the Pangeo project is an ecosystem of mutually compatible
+Geoscience python packages which follow open-source best practices.
+These practices are well established across the scientific python community.
+
+
+### General Best Practices for Open Source
+
+1. Open source license
```

Comment:
This should probably link to either [Jake VanderPlas' article on licensing scientific code](http://www.astrobetter.com/blog/2014/03/10/the-whys-and-hows-of-licensing-scientific-code/) or [these more general guidelines](https://choosealicense.com/).